framework/src/ics/BoundingBoxIC.C

```diff
@@ -46,16 +50,30 @@ BoundingBoxIC::BoundingBoxIC(const InputParameters & parameters)
     _inside(getParam<Real>("inside")),
     _outside(getParam<Real>("outside")),
     _bottom_left(_x1, _y1, _z1),
-    _top_right(_x2, _y2, _z2)
+    _top_right(_x2, _y2, _z2),
+    _int_width(getParam<Real>("int_width"))
 {
 }
 
 Real
 BoundingBoxIC::value(const Point & p)
 {
-  for (unsigned int i = 0; i < LIBMESH_DIM; i++)
-    if (p(i) < _bottom_left(i) || p(i) > _top_right(i))
-      return _outside;
+  if (_int_width <= 0.0)
```

Comment:
There is no reason to permit negative interface widths, is there? I suggest making the parameter range checked with `"int_width >= 0.0"` and setting this to `== 0.0`